stripper/ze_ffxii_feywood_b6_3k

```diff
@@ -0,0 +1,430 @@
+		"OnNewGame" "lvl3_Boss_Push1AddOutputmaxs 16 840 1281-1"
+		"OnNewGame" "lvl3_Boss_HP_Add_BxzlAddOutputsolid 20.5-1"
+		"OnNewGame" "lvl3_Boss_HP_Add_BxzlAddOutputmins -8 -840 -1281-1"
+		"OnNewGame" "lvl3_Boss_HP_Add_BxzlAddOutputmaxs 8 840 1281-1"
+		"OnNewGame" "lv4_BossCage_Teleport1AddOutputsolid 20.5-1"
+		"OnNewGame" "lv4_BossCage_Teleport1AddOutputmins -77 -77 -681-1"
+		"OnNewGame" "lv4_BossCage_Teleport1AddOutputmaxs 77 77 681-1"
+		"OnNewGame" "lv4_BossCage_Teleport2AddOutputsolid 20.5-1"
+		"OnNewGame" "lv4_BossCage_Teleport2AddOutputmins -77 -77 -681-1"
+		"OnNewGame" "lv4_BossCage_Teleport2AddOutputmaxs 77 77 681-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport1AddOutputsolid 20.5-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport1AddOutputmins -200 -8 -2561-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport1AddOutputmaxs 200 8 2561-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport2AddOutputsolid 20.5-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport2AddOutputmins -200 -8 -2561-1"
+		"OnNewGame" "lv4_RotatingAxe_Teleport2AddOutputmaxs 200 8 2561-1"
+	}
+}
+
+//add lvl3 dragon hp detect
+filter:
+{
+		"targetname" "lvl3_Boss_HP_Add"
+		"classname" "trigger_multiple"
+}
+modify:
+{
+	match:
+	{
+		"targetname" "lvl3_Boss_Push"
+		"classname" "trigger_push"
+	}
+	delete:
+	{
+		"OnStartTouch" "lvl3_Boss_HP_AddFireUser201"
+	}
+}
+modify:
+{
+	match:
+	{
+		"targetname" "lvl3_Boss_Killed"
+		"classname" "logic_relay"
+	}
+	insert:
+	{
+		"OnTrigger" "lvl3_Boss_HP_Add_BxzlFireUser10-1"
+		"OnTrigger" "lvl3_Boss_PushFireUser10-1"
+	}
+}
+add:
+{
+	"targetname" "lvl3_Boss_Push1"
+	"StartDisabled" "0"
+	"speed" "350"
+	"spawnflags" "1"
+	"pushdir" "0 0 0"
+	"origin" "2800 5176 3344"
+	"filtername" "Humans_Filter"
+	"alternateticksfix" "0"
+	"classname" "trigger_push"
+	connections
+	{
+		"OnStartTouch" "lvl3_Boss_HP_AddEnable01"
+		"OnStartTouch" "Boss_HPbar_CounterEnable01"
+		"OnStartTouch" "Boss_HP_2Enable01"
+		"OnStartTouch" "Boss_HPEnable01"
+	}
+}
+add:
+{
+	"targetname" "lvl3_Boss_HP_Add_Bxzl"
+	"StartDisabled" "0"
+	"spawnflags" "1"
+	"origin" "2804 5176 3344"
+	"nodmgforce" "0"
+	"filtername" "Humans_Filter"
+	"damagetype" "0"
+	"damagemodel" "0"
+	"damagecap" "20"
+	"damage" "-2"
+	"classname" "trigger_hurt"
+	connections
+	{
+		"OnStartTouch" "Boss_HPAdd220-1"
+		"OnUser1" "!selfDisable0-1"
+	}
+}
+
+//ADD Another push at the end boss in level 3
+add:
+{
+	"targetname" "lvl3_finalpush"
+	"StartDisabled" "1"
+	"speed" "350"
+	"spawnflags" "1"
+	"pushdir" "0 270 0"
+	"origin" "10641 6390 2673"
+	"filtername" "Humans_Filter"
+	"FallingSpeedThreshold" "-150"
+	"alternateticksfix" "0"
+	"classname" "trigger_push"
+}
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "lvl3_Boss_Killed"
+	}
+	insert:
+	{
+		"OnTrigger" "lvl3_finalpushEnable0-1"
+	}
+}
+
+//change item's particle in order to make owner can be hidden by other player
```

Comment:
What is this change doing? The comment doesn't explain it well.

Reply:
I am pretty sure it was for 93x's hide plugin to work concerning players with items. Unneeded on GFL so it has been removed.